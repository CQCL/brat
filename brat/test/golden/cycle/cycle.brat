import cycle_aux

main :: Unit
main = <>
