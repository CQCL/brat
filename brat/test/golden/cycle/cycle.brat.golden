ret > ExitFailure 1
err > Error in cycle:
err >   Cycle detected in imports: cycle is reachable from cycle_aux
err >
err >
